store/sqlstore/upgrade_test.go

```diff
@@ -13,6 +13,16 @@ import (
 	"github.com/mattermost/mattermost-server/v5/store"
 )
 
+func TestStoreUpgradeDotRelease(t *testing.T) {
+	StoreTest(t, func(t *testing.T, ss store.Store) {
+		sqlStore := ss.(*SqlStore)
+		saveSchemaVersion(sqlStore, "5.33.1")
+		err := upgradeDatabase(sqlStore, CurrentSchemaVersion)
```

Comment:
Would prefer to have a static string here rather than something which changes. 0/5

Reply:
I don't think it would work though because the final version gets set to whatever the latest upgrade version is.

Reply:
Oh yeah my bad, this is the top level function. I thought you were testing `shouldPerformUpgrade` specifically.